docs/application/native/guides/app-management/app-controls.md

```diff
@@ -29,6 +29,10 @@ The main App Control API features are:
 
   You can [set the launch mode](#group) when launching an application through an application control. The launch mode determines whether the application is launched in its own group, or as a sub application within an existing group.
 
+- Handle application control actions
+
+  You can handle each application control action when getting a launch request.
+
 You can take advantage of the Tizen [base application functionalities](#common) through the application control feature.
 
 
```

Comment:
Line 42:
Kindly rephrase the line as "You can launch an application with application control using the following launch types:"

Line 48:
Please replace 'matched' with 'match' and 'result' with 'as result'.

Line 59:
Kindly rephrase the line as "1.	Ensure, you determine which application can be launched such that the application launcher framework compares the operation, URI, and MIME type conditions."

Line 70:
Kindly rephrase the line as "To launch an application with the application control, you should create a launch request. To launch application with selected conditions you must create and add the app_control handle. You can add the following information"

Line 80:
Please move the line to 'Note' section mentioned for 'Operation'. As it feels disconnected while reading.

Line 82:
Definition of URI does not seems to be accurate, as I checked the information here: https://searchmicroservices.techtarget.com/definition/URI-Uniform-Resource-Identifier. Can you please update it with the accurate definition.

Line 172:
Here quotes are used at "resolving the application control", are you trying to emphasize or highlight this phrase? Kindly explain as while reading it did not register actually,

Line 186:
Kindly replace 'their' with 'your'.

Line 188:
Kindly rephrase the line as "...The detailed conditions for matches are as follows:"

Line 191 to Line 194 and Line 196 to Line 198 :
In this bullet list keep first letter in small case

Line 197:
Kindly replace 'has' with 'have'

Line 280:
Kindly rephrase the line as "...when the application launcher framework receives and resolves a launch request..."

Line 282 (Global Comment):
Please drop the words 'the' and 'function' while writing a function name. Suggested rephrase "..to the application framework by calling ui_app_main()"

Line 342:
Please write 4 as 'four' there and rephrase the line as "...the values from the application framework, and these values must remain unchanged. The third parameter is a state transition handler that is responsible for managing the state transitions the application goes through, while it is running. The fourth parameter is application data that is passed to each state handler."

Line 344:
Please remove 'first' and rephrase as "When the `ui_app_main()` is invoked....". Also apply the Global Comment mentioned earlier to remove function word while writing a function name.

Line 349, 350, 351:
Kind rephrase the lines as 
"Just after the application enters the main loop, the application framework calls the 'app_control_cb()' callback.
The callback is passed to the app_control handle containing the reason why the application was launched.
 
For example, the application can be launched to open a file to handle the request that is sent by other application. The application is always responsible for checking the content of the app_control handle and respond appropriately. The content of the app_control handle can be empty, if the application is launched by the user from the launcher."

Line 397:
Kindly re-write the "(callee)" as "(known as callee in this example)" and "(caller)" as "(known as caller in this example)"

Line 454:
Kindly rephrase as "...to launch your application and use its application features through..."

Line 464:
Please drop 'a' in the sentence and rephrase as "Any application requesting launch of your application must either..."

Line 472:
Kindly rephrase as "...function is available, where '\*' matches an arbitrary..."

Line 474:
Kindly rephrase the line as "You can specify the control information from [application project settings](../../tutorials/process/setting-properties.md#manifest)"

Line 488:
Kindly rephrase the line as "The caller launch mode means that the application is launched as a sub application that belong to the same group as the caller application, which is launching the application."

Line 490:
Kindly rephrase the line as "... If the launch mode is set to `caller`, the application that calls the app control can define the launch mode for the called application using `app_control_set_launch_mode()`."

Reply:
> Line 82:
> Definition of URI does not seems to be accurate, as I checked the information here: https://searchmicroservices.techtarget.com/definition/URI-Uniform-Resource-Identifier. Can you please update it with the accurate definition.

=> Generally, URI means Uniform Resource Identifier. I understand that the author wrote in the meaning of the data of app control.

Reply:
Done